tensorflow/core/BUILD

```diff
@@ -3571,6 +3571,7 @@ tf_cuda_library(
         ":graph",
         ":lib",
         ":lib_internal",
+        "//tensorflow/core/nccl:nccl_lib",
```

Comment:
nit: this would break sanity checks. could you sort the dependency list properly?